### PR TITLE
add accessibility day bar

### PR DESCRIPTION
**Pull request by dstodolny**
WIP do not merge

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by gregdorward** [comment]
- [ ] Reminder once the grey scale is working on all other sites, we have an issue with iOS 9, in that it defaults to colour and cannot be transitioned to grey scale

**Comment by gregdorward** [comment]
Not sure if this is useful for the above issue - http://stackoverflow.com/questions/25238120/css-webkit-filter-grayscale0-dont-work-in-ie-10-and-safari

File: site/index.ejs [hunk 1]
```diff
@@ -39,8 +39,86 @@
         ga('create', 'UA-16654919-1', 'auto');
         ga('send', 'pageview');
     </script><% } %>
+    <style>
+      .accessibilityDay {
+        filter: grayscale(100%);
```
Comment (gregdorward) [inline site/index.ejs]: Could we get the following browser specific filters added to this please?

        -webkit-filter: grayscale(100%);
        -moz-filter: grayscale(100%);
        -ms-filter: grayscale(100%);
        -o-filter: grayscale(100%);

I can't seem to find a simple solution for IE10 and IE11. There are some convoluted workarounds out there, but they all looked a little bit complex compared to how these have been applied

File: site/index.ejs [hunk 2]
```diff
@@ -39,8 +39,86 @@
...
+          button.innerHTML = colourOffButtonText;
+        }
+      }
+    </script>
   </head>
-  <body>
+  <body onload="initialiseAccessibilityBar()">
+    <div class="accessibilityBar">
+      <button id="accessibilityDayButton" onClick="toggleGreyscale()">Switch back to colour</button>
+      <p id="accessibilityDayDescription">What’s black and white and read all over? Us. We’re grey scale to promote Global Accessibility Awareness Day.</p>
```
Comment (gregdorward) [inline site/index.ejs]: Feedback from Sam - grey scale is actually one word 'greyscale'. I've looked this up and I think he's right, so can we get this changed as well please?

File: site/index.ejs [hunk 3]
```diff
@@ -39,8 +39,90 @@
...
+      }
+
+      .accessibilityBar {
+        width: 100%;
+        padding: 20px;
+        background-color: #22d59b;
+        text-align: center;
+      }
+
+      .accessibilityBar button {
```
Comment (Xanndarr) [inline site/index.ejs]: Still has default chrome on click. Small but doesn't match behaviour of the other buttons:

screen shot 2017-05-17 at 12 10 20

**Pull request merged**